extensions/search-npm/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "search-npm",
+  "title": "Search npm",
+  "description": "",
```

Can you please provide a description? Again, this is used in the store.
```suggestion
  "description": "Search for npm package information from npms.io",
```